components/ResourceTable.tsx

```diff
@@ -1,47 +1,148 @@
-import React, { useState } from "react";
+import React, {useEffect, useState} from "react";
 import Link from "next/link";
 import BrTable from "./brickroom/BrTable";
-import QrCodeButton from "./brickroom/QrCodeButton";
+import {gql, useQuery} from "@apollo/client";
+import devLog from "../lib/devLog";
+import BrLoadMore from "./brickroom/BrLoadMore";
+import Spinner from "./brickroom/Spinner";
 
 const truncate = (input: string, max: number) => input.length > max ? `${input.substring(0, max)}...` : input;
 
+const FETCH_INVENTORY = gql`query($first: Int, $after: ID, $last: Int, $before: ID, $filters:EconomicResourceFilterParams ) {
+  economicResources(first: $first after: $after before: $before last: $last filter: $filters) {
+    pageInfo {
+      startCursor
+      endCursor
+      hasPreviousPage
+      hasNextPage
+      totalCount
+      pageLimit
+    }
+    edges {
+      cursor
+      node {
+        conformsTo {
+          id
+          name
+        }
+        currentLocation {id name mappableAddress}
+        id
+        name
+        note
+        metadata
+        okhv
+        repo
+        version
+        licensor
+        license
+        primaryAccountable {id name note}
+        custodian {id name note}
+        accountingQuantity {hasUnit{id label symbol} hasNumericalValue}
+        onhandQuantity {hasUnit{id label symbol} hasNumericalValue}
+      }
+    }
+  }
+}
+`
+
 
-const ResourceTable = ({ resources }: { resources: Array<any> }) => {
+const ResourceTable = ({filter}: { filter: any }) => {
     const resourcesHead = ['Resource', 'Source', 'License', 'Version']
+
+    // @ts-ignore
+    const {loading, data, error, fetchMore, variables, refetch} = useQuery(FETCH_INVENTORY, {
+        variables: {
+            last: 10,
+            filters: filter
+        }
+    })
+    console.error(error);
```

Comment:
maybe you want to have also a devError... or this should be triggered in a `catch`? each time `error` is `null` and or changes this prints something, is this wanted on purpose?